CONTRIBUTING.md

```diff
@@ -167,7 +190,12 @@ sudo apt install \
 To install `podman`, you can find the appropriate guide for your respective system:
 [Install Podman](https://podman.io/getting-started/installation)
 
+<<<<<<< HEAD
 Next you'll want to clone the repository and get into it:
+=======
+Next you'll want to clone the repository and cd into it:
+
+> > > > > > > 78c002d1 (initial setup-continued)
```

Comment:
will these lines render? It looks like just some left overs from the merge conflict

Reply:
left over, whoops, ptal again, pushed a fix, thanks